src/io/camera.js

```diff
@@ -3,14 +3,12 @@
 module.exports = function Camera(options) {
   var canvas, 
       ctx;
-
+  options.webcamVideoSelector = options.webcamVideoSelector || 'webcamVideoEl';
+  options.webcamVideoEl = document.getElementById(options.webcamVideoSelector);
+  
   // Initialize getUserMedia with options
   function initialize() {
-    getUserMedia(webRtcOptions, success, deviceError);
```

Comment:
For example here!

Reply:
```diff
  function initialize() {
+     if (options.version ==1){
    getUserMedia(webRtcOptions, success, deviceError);
+   }else{
}
```

Reply:
I think i get it now .

Reply:
I believe what @Forchapeatl [just now updated](https://github.com/publiclab/infragram/pull/442/files#r939453098) should work:

```javascript
  function initialize() {
+     if (options.version == '1'){
    getUserMedia(webRtcOptions, success, deviceError);
+   }else{
}
```

Reply:
Yes exactly!!

Reply:
Would you mind testing it out, then let me know when you're ready!

Reply:
And be careful, JavaScript has some odd behaviors when comparing different types. I think the flag is an integer, not a string, so double check!